warcry_proj/README.txt

```diff
@@ -0,0 +1,22 @@
+warcry_proj README
+==================
+
+Getting Started
+---------------
+
+- Login to Vagrant
+* vagrant up
+* vagrant ssh
+
+- Ensure the bootstrap script is run to configure and install all required packages
+* sudo script/bootstrap vagrant
```

Comment:
I would just call it project.yml

The example project it an extension of the greater whole... normally you wouldn't put anything from the "examples" in the main line unless they affect all projects. In theory, in this case, anything we put in here is part of the boilerplate, and all example projects would do the same thing, so it's OK.